openapi.json

```diff
@@ -8669,7 +8772,7 @@
         "content": {
           "application/json": {
             "schema": {
-              "$ref": "#/components/schemas/requestBody"
+              "$ref": "#/components/schemas/Json"
```

Comment:
did you verify it's not breaks anything else (it's the cli after all ;) )

Reply:
sure, there was not such schema)